.travis.yml

```diff
@@ -17,8 +17,7 @@ env:
     - BINARYBUILDER_AUTOMATIC_APPLE=true
 sudo: required
 before_script:
-  - julia -e 'import Pkg; Pkg.add(Pkg.PackageSpec(url="https://github.com/JuliaPackaging/BinaryBuilder.jl", rev="master"))'
-  - julia -e 'import Pkg; Pkg.add("BinaryProvider")'
+  - julia -e 'using Pkg; pkg"add BinaryProvider"; pkg"add BinaryBuilder#master"; Pkg.build()'
 script:
```

Comment:
It is very much preferred to use the "API" interface of Pkg in scripts than the REPL interface.

Reply:
Ok, but note that this is from BinaryBuilder wizard: https://github.com/JuliaPackaging/BinaryBuilder.jl/blob/bb4fd5e769093a9ebcff8d4377ed6c351d115019/src/wizard/deploy.jl#L100